singer_sdk/helpers/_typing.py

```diff
@@ -489,6 +506,9 @@ def _conform_primitive_property(  # noqa: PLR0911
         if is_boolean_type(property_schema):
             bit_representation = elem != b"\x00"
             return bit_representation
+        # if property_schema.get("contentEncoding"):
```

Comment:
```suggestion
```

Reply:
Thank you for pointing this out 🙏.  It has been removed.